pom.xml

```diff
@@ -297,7 +296,8 @@
 				<configuration>
 					<signature>
 						<groupId>org.codehaus.mojo.signature</groupId>
-						<artifactId>java1${java.level}</artifactId>
+						<artifactId>java18</artifactId>
```

Comment:
Per https://github.com/mojohaus/animal-sniffer/issues/62, keeping animal-sniffer when building with Java 11 does not really make sense.

Reply:
So, we would be able to remove this configuration ?

Reply:
This is what I did in https://github.com/gitlab4j/gitlab4j-api/pull/946 (keeping you are co-author of the change)